Resources/Prototypes/Entities/Structures/Dispensers/base_structuredispensers.yml

```
@@ -77,3 +77,7 @@
   - type: StaticPrice
     price: 1000
   - type: WiresPanel
+  - type: EmptyOnMachineDeconstruct
+    containers:
+      - storagebase
+      - beakerSlot
```

```suggestion
    containers:
    - storagebase
    - beakerSlot
```
[yaml convention](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions)